app/views/shared/_banner.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="banner"">
+    <nav>
```

Comment:
Pls correct me if im wrong, but I think the Navbar should have its own partial which will be rendered in the application.html.erb

Reply:
Not necessarily in the layout itself because there might be some pages where we don't want the navbar.
The admin interface wouldn't need it for example.

But I do agree moving the navbar in a reusable component would be better

Reply:
you mean to only render it in the homepage right? not in the pages where it wont be needed.

Reply:
Yes if Alex makes a partial out of the navbar then we can render it only in the pages we want and skip the others 🙂

Reply:
Hello gentlemen! I totally agree that the navbar should be a partial. I just got home from work so i wont work on it today but tomorrow i have office day so i can deal with it ! Have a nice day!